tasks/bootstrap_local_vm/02_create_local_vm.yml

```diff
@@ -15,7 +15,13 @@
   - name: Initial tasks
     block:
       - name: Get host unique id
-        shell: if [ -e /etc/vdsm/vdsm.id ]; then cat /etc/vdsm/vdsm.id; else dmidecode -s system-uuid; fi;
+        shell: |
+           if [ -e /etc/vdsm/vdsm.id ];
```

Comment:
Last thing, can you please indent it by two chars instead of three?

Reply:
Done